moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -352,4 +360,47 @@ class CartTrajectoryPoint : public TrajectoryPoint
   Eigen::Affine3d pose_;
 };
 
+class addTrajPointException: public std::runtime_error
```

Comment:
Are you suggesting that exceptions should be defined for the whole namespace and defining them hierarchically for each method is not a good idea?